label_studio/tasks/models.py

```diff
@@ -847,7 +847,7 @@ def delete(self, *args, **kwargs):
 
 
 class Prediction(models.Model):
-    """ML backend predictions"""
+    """ML backend / prompts predictions"""
```

Comment:
nit: can you capitalize to make it clear we mean the "Prompts" project, not the general idea of prompts (I know the context is pretty clear, just want it to be stupidly obvious)

Reply:
Will do